cmake/treedata/android/subdirs.txt

```diff
@@ -1,4 +1,5 @@
 xbmc/cores/RetroPlayer/process/android  cores/RetroPlayer/process/android
+xbmc/drm/platform/android               drm/platform/android
```

Comment:
it's be better to move platform specific code to platform folder i.e. `xbmc/platform/android/drm`